### PR TITLE
Internship announcement blog post

### PR DESCRIPTION
**Pull request by douwekiela**
First version of the blog post announcing our internship positions for 2023.

TODO:
* We need a thumbnail
* We need to add the correct links to the Workable job listings

### CODE REVIEW

File: _blog.yml [hunk 1]
```diff
@@ -1474,4 +1474,13 @@
...
   - clip
   - stable-diffusion
-  - dalle
\ No newline at end of file
+  - dalle
+
+- local: internships-2023
+  title: "We are hiring interns!"
+  author: douwekiela
+  thumbnail: /blog/assets/internships-2023/thumbnail.jpg
```
Comment (osanseviero) [inline _blog.yml:1482]: The thumbnail is a png I think? 
But the png says invalid image source too

File: internships-2023.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+* [Model Forgetting Internship](https://huggingface.workable.com/jobs/2826105), investigating model forgetting using state of the art “unlearning” methods.
+* [Multimodal AI Internship](https://huggingface.workable.com/jobs/2826106), working at the intersection of natural language processing and computer vision for the next generation of large multimodal models.
+* [OCR Internship](https://huggingface.workable.com/jobs/2832512), incorporating textual information directly into multimodal models.
+* [Video Pretraining Internship](https://huggingface.workable.com/jobs/2832513), exploring pretraining for the video modality.
+* [Retrieval Augmentation Internship](https://huggingface.workable.com/jobs/2832515), investigating “semiparametric” retrieval augmentation methods for more consistent, more interpretable, and more efficient language models.
+* [Embodied AI Internship](https://huggingface.workable.com/jobs/2797383), working with the Embodied AI team on reinforcement learning in simulators.
+* [Benchmarking Transformers Internship](https://apply.workable.com/huggingface/j/17BC19A84B), deploying and benchmarking transformer models for both training and inference.
```
Comment (LysandreJik) [inline internships-2023.md:57]: I think this should go in "other" rather than science, right?

File: internships-2023.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+* [Social Impact Evaluation Internship](https://huggingface.workable.com/jobs/2826102), developing a technical framework for assessing the overall social impact of generative ML models.
+* [Gradio Internship](https://huggingface.workable.com/jobs/2832520), working with the Gradio team on expanding the ecosystem.
+* [AI Art Tooling Internship](https://huggingface.workable.com/jobs/2832517), bridging the AI and art worlds by building tooling to empower artists.
+
+Locations vary on a case-by-case basis and if the internship host has a location preference, this will be indicated on the job listing.
+
+## How to Apply
+
+You can apply directly for each position through our [job portal](https://huggingface.workable.com/). Click on the positions above to be taken directly to the application form.
```
Comment (LysandreJik) [inline internships-2023.md:70]:
```suggestion
You can apply directly for each position through our [job portal](https://huggingface.workable.com/). Click on the positions above to be taken directly to the application form.

Please make sure to complete the short submission at the end of the application form when applying. You'll need to create a Hugging Face account for that.
```

**Review by LysandreJik — APPROVED** [review]
LGTM!

File: _blog.yml [hunk 4]
```diff
@@ -1474,4 +1474,13 @@
   - text to image
   - clip
   - stable-diffusion
-  - dalle
\ No newline at end of file
+  - dalle
+
+- local: interns-23
```
Comment (julien-c) [inline _blog.yml:1479]: interns-2023 would be clearer no?

Comment (douwekiela) [reply _blog.yml:1479]: done, thanks!

**Pull request merged**